src/vivarium_csu_zenon/data/loader.py

```diff
@@ -473,21 +439,66 @@ def load_ikf_exposure(key: str, location: str) -> pd.DataFrame:
     # normalize so all categories sum to 1
     cols = list(set(data.columns).difference(vi_globals.DRAW_COLUMNS + ['parameter']))
     sums = data.groupby(cols)[vi_globals.DRAW_COLUMNS].sum()
-    data = (data.groupby('parameter')
+    data = (data
```

Comment:
Why are you doing this? Didn't you already normalize the data in line 424?

Reply:
Oh, this was just a formatting thing, didn't actually touch the logic.  Should be a no-op, I agree.  You can drop it if you like.